reakit/reakit
# feat: improve separator aria return

Adds extra checks to Separator Component as discussed in the below issue,
Closes #721 

Added Storybook example to demonstrate the behaviour,

Path: [packages/reakit/src/Separator/__examples__/Seperator/index.tsx](https://github.com/reakit/reakit/compare/next...feat/separator-update?expand=1#diff-9704763ecd7170b58cbe7b2e5da305a1)

![image](https://user-images.githubusercontent.com/39694575/93661467-abe12c80-fa75-11ea-8353-dec324c4f9fd.png)

![image](https://user-images.githubusercontent.com/39694575/93661474-b996b200-fa75-11ea-95ec-ec314a6f3cab.png)

**How to test?**

Run `yarn storybook` to see the `Separator Example` story and test it.
Added tests myself -> [packages/reakit/src/Separator/__tests__/Separator-test.tsx](https://github.com/reakit/reakit/compare/next...feat/separator-update?expand=1#diff-3a64a6bea74946a412197176a31a3141)

**Does this PR introduce breaking changes?**

May be 🤔 , used `as` to acheive the behaviour.
Passed down `as` from `createComponent` to `useHook`
